src/fidesops/service/privacy_request/request_runner_service.py

```diff
@@ -22,237 +23,219 @@
-                        privacy_request=privacy_request,
-                        policy=policy,
-                        graph=dataset_graph,
-                        connection_configs=connection_configs,
-                        identity=identity_data,
-                    )
+    for rule in policy.get_rules_for_action(action_type=ActionType.access):
+        if not rule.storage_destination:
+            raise common_exceptions.RuleValidationError(
+                f"No storage destination configured on rule {rule.key}"
+            )
+        target_categories: Set[str] = {target.data_category for target in rule.targets}
+        filtered_results = filter_data_categories(
+            access_result,
+            target_categories,
+            dataset_graph.data_category_field_mapping,
+        )
+        logging.info(
+            f"Starting access request upload for rule {rule.key} for privacy request {privacy_request.id}"
+        )
+        try:
+            upload(
+                db=session,
+                request_id=privacy_request.id,
+                data=filtered_results,
+                storage_key=rule.storage_destination.key,
+            )
+        except common_exceptions.StorageUploadError as exc:
+            logging.error(
+                f"Error uploading subject access data for rule {rule.key} on policy {policy.key} and privacy request {privacy_request.id} : {exc}"
+            )
+            privacy_request.status = PrivacyRequestStatus.error
 
-                    self.upload_access_results(
-                        session, policy, access_result, dataset_graph, privacy_request
-                    )
 
-                if policy.get_rules_for_action(action_type=ActionType.erasure):
-                    # We only need to run the erasure once until masking strategies are handled
-                    run_erasure(
-                        privacy_request=privacy_request,
-                        policy=policy,
-                        graph=dataset_graph,
-                        connection_configs=connection_configs,
-                        identity=identity_data,
-                        access_request_data=get_cached_data_for_erasures(
-                            privacy_request.id
-                        ),
-                    )
+@celery_app.task()
+def run_privacy_request(
+    privacy_request_id: str,
+    from_webhook_id: Optional[str] = None,
+    from_step: Optional[str] = None,
+) -> None:
+    # pylint: disable=too-many-locals
+    """
+    Dispatch a privacy_request into the execution layer by:
+        1. Generate a graph from all the currently configured datasets
+        2. Take the provided identity data
+        3. Start the access request / erasure request execution
+        4. When finished, upload the results to the configured storage destination if applicable
+    """
+    logger.info(f"Executing request with ID: {privacy_request_id}")
```

Comment:
Now that you have logging working, is this still needed, given that we log a very similar thing on L149?